pytroll_collectors/s3stalker.py

```diff
@@ -93,17 +189,33 @@ def set_last_fetch(timestamp):
     DatetimeHolder.last_fetch = timestamp
 
 
+def get_last_fetch():
+    """Get the last fetch time."""
+    return DatetimeHolder.last_fetch
+
+
+def create_messages_for_recent_files(bucket, config):
+    """Create messages for recent files and return."""
+    logger.debug("Create messages for recent files...")
+    time_back = config['timedelta']
+    logger.debug("time_back = %s", str(time_back))
+    subject = config['subject']
+    pattern = config.get('file_pattern')
+    with sleeper(2.5):
```

Comment:
Is it important that the code block should take at least 2.5 seconds to process?

Reply:
Comments/reasoning for this?

Reply:
Not sure about the exact amount of `2.5` how that was achieved. Do you remember @TAlonglong ?
I can move it into a configuration item, would that be okay?

Reply:
Actually git-blame says it was added by @mraspaud